### PR TITLE
fix some linter errcheck for tae

### PR DESCRIPTION
**Pull request by XuPeng-SH**
**What type of PR is this?**

- [ ] API-change
- [ ] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [x] Code Refactoring

**Which issue(s) this PR fixes:**

issue #2523 

**What this PR does / why we need it:**

Not Available

**Special notes for your reviewer:**

Not Available

**Additional documentation (e.g. design docs, usage docs, etc.):**

Not Available

### CODE REVIEW

File: pkg/vm/engine/tae/tables/insert.go [hunk 1]
```diff
@@ -59,8 +59,8 @@ func newInsertInfo(rwlocker *sync.RWMutex, maxTs uint64, capacity uint32) *inser
 
 func (info *insertInfo) RecordTxnLocked(offset uint32, txn txnif.TxnReader) {
 	pos := uint32(info.offsets.Length())
-	info.offsets.Append(1, []uint32{offset})
-	info.ts.Append(1, []uint64{txn.GetCommitTS()})
+	_ = info.offsets.Append(1, []uint32{offset})
```
Comment (fengttt) [inline pkg/vm/engine/tae/tables/insert.go:62]: Can you turn this _ into an assert or panic?

File: pkg/vm/engine/tae/tables/node.go [hunk 2]
```diff
@@ -212,7 +212,7 @@ func (node *appendableNode) OnUnload() {
 	node.data.Close()
 	node.data = nil
 	if needCkp {
-		node.block.scheduler.ScheduleScopedFn(nil, tasks.CheckpointTask, node.block.meta.AsCommonID(), node.block.CheckpointWALClosure(ts))
+		_, _ = node.block.scheduler.ScheduleScopedFn(nil, tasks.CheckpointTask, node.block.meta.AsCommonID(), node.block.CheckpointWALClosure(ts))
```
Comment (fengttt) [inline pkg/vm/engine/tae/tables/node.go:215]: Same here.

**Review by fengttt — COMMENTED** [review]
For those not in test, please do not use _ to mask off error.   Either return it to caller, or, panic, or, add a comment, explicit explain why it is OK to ignore the error here.

Thank you.

File: pkg/vm/engine/tae/tables/insert.go (hunk 1 above)
Comment (XuPeng-SH) [reply pkg/vm/engine/tae/tables/insert.go:62]: Ok. I'll add some comments for _

**Review by LeftHandCold — APPROVED** [review]
lgtm

**Pull request merged**